apps/test/unit/templates/teacherDashboard/SectionActionDropdownTest.js

```diff
@@ -133,4 +144,36 @@ describe('SectionActionDropdown', () => {
     );
     expect(wrapper).to.contain('Restore Section');
   });
+
+  // Uncomment for launch
+  // it('sends selected user to the new edit page', () => {
+  //   const wrapper = shallow(
+  //     <SectionActionDropdown
+  //       {...DEFAULT_PROPS}
+  //       sectionData={sections[3]}
+  //       userId={1539340}
+  //     />
+  //   );
+  //   const sectionId = wrapper.instance().props.sectionData.id;
+  //   const expectedUrl = '/sections/' + sectionId + '/edit';
+  //   expect(wrapper).to.contain('Edit Section Details');
+  //   expect(wrapper.find('.edit-section-details-link').props().href).to.equal(
+  //     expectedUrl
+  //   );
+  // });
+
+  // this test can be deleted once we fully switch over to the new flow
+  it('sends user to the old edit page', () => {
+    const wrapper = shallow(
+      <SectionActionDropdown
+        {...DEFAULT_PROPS}
+        sectionData={sections[3]}
+        userId={1539347}
+        handleEdit={onEditFunction}
+      />
+    );
+    expect(wrapper).to.contain('Edit Section Details');
+    wrapper.find('.edit-section-details-link').simulate('click');
+    expect(onEditFunction).to.have.been.calledOnce;
```

Comment:
Just making sure I understand- this handleEdit function will only be called if a user is using the old flow, yes? And otherwise they just get an href button. If so, then this test is great!

Reply:
Correct!